earnings.sh

```diff
@@ -0,0 +1,32 @@
+#Script prints the following information for a given dataset:
+#Unique gender-yearsExperience combinations
+#Gender, years of Experience, wage for highest earner
+#Gender, years of experience, wage for lowest earner
+#Number of women in top 10 earners
+#Effect of graduating college on the minumum wage of earners
+
+#Usage: bash earnings.sh filename.csv
+#1:
+cat $1 | sed 's/,/ /g' | cut -d ' ' -f 1,2 | uniq | sort -n -k 1 | sort -n -k 2 | uniq > output_ex4.txt
+
```

Comment:
Sort with gender first then by yearExperience. It should be: 
cat $1 | sed 's/,/ /g' | cut -d ' ' -f 1,2 | uniq | sort -n -k 2 | sort -n -k 1 | uniq > output_ex4.txt

Or:
cat wages.csv | grep -v gender | cut -d , -f 1,2 | sort -u | tr ',' ' ' | sort -k1,1d -k2,2n > categories.txt